crates/metastore/src/local.rs

```diff
@@ -34,11 +34,13 @@ pub async fn start_inprocess(
     let (client, server) = tokio::io::duplex(1024);
 
     tokio::spawn(async move {
-        Server::builder()
+        if let Err(e) = Server::builder()
             .add_service(MetastoreServiceServer::new(Service::new(store)))
             .serve_with_incoming(futures::stream::iter(vec![Ok::<_, MetastoreError>(server)]))
             .await
-            .unwrap()
+        {
+            eprintln!("internal error: {}", e);
+        }
```

Comment:
I originally added this when debugging the metastore issue mentioned on 12/11. I think it makes sense to have some logging if the handle panics. Happy to remove if we don't want it though.